regression/verilog/SVA/case1.desc

```diff
@@ -0,0 +1,8 @@
+KNOWNBUG
+case1.sv
+--bound 20
+^EXIT=10$
+^SIGNAL=0$
+--
+^warning: ignoring
+--
```

Comment:
Can you please add a comment explaining what is going wrong here?

Reply:
Added the implementation.